### ISSUE TITLE
Motion detected when opening home app.

### ISSUE THREAD

**Issue by NajibNour** [lucavb/homebridge-hc-sr501#13]
Hello,
Hope everyone is doing well. I am currently having an issue were when I open the home app motion gets triggered. The room would be fully empty with no movement what so ever.

Any ideas on why?

Let me know!

Thanks.

**Comment by lucavb** [comment]
Hang on. You are saying that the "sensor" gets triggered when you open the app?

**Comment by NajibNour** [comment]
Yes sir, I have diagnosed a little further, when I open the home app and scroll down to my nest cam, the sensor triggers as motion once it has finished loading the new snapshot for the nest camera. Not sure if the false trigger is related to it but that is something I have noticed.

**Comment by lucavb** [comment]
And if you switch of this camera temporarily?

**Comment by NajibNour** [comment]
After removing all nest cameras, the issue is no longer present. After adding one nest cam the issue represents itself.

**Comment by lucavb** [comment]
some infrared stuff? that sounds rather strange.

**Comment by NajibNour** [comment]
I have used an AM312 sensor instead and it is stable. The only issue I am having now is that there is no constant motion trigger. Looking into changing the code to my use case.

**Comment by lucavb** [comment]
What do you mean by no constant motion trigger?

**Comment by NajibNour** [comment]
As the HC-SR501 has a time knob that sets how long that the output will remain HIGH after detection. The AM134 sensor does not have that option so I am trying to see if I can develop that into the code with my limited knowledge in coding.
